xml/System.IO.MemoryMappedFiles/MemoryMappedFile.xml

```diff
@@ -767,6 +767,8 @@ Process C says: True
         <exception cref="T:System.ArgumentOutOfRangeException">
           <paramref name="capacity" /> is less than or equal to zero.</exception>
         <related type="Article" href="https://msdn.microsoft.com/library/a483d1b5-64aa-45b6-86ef-11b859f7f02e">Memory-Mapped Files</related>
```

Comment:
It would be good to add the exceptions together like this and we also use a simplified paramref tag.

```suggestion
        <exception cref="T:System.ArgumentNullException">
          <paramref name="mapName" /> is <see langword="null" />.</exception>
        <related type="Article" href="https://msdn.microsoft.com/library/a483d1b5-64aa-45b6-86ef-11b859f7f02e">Memory-Mapped Files</related>
```

Reply:
This is an interesting catch. This exception was obtained from the artifacts/bin/docs/System.IO.MemoryMappedFiles.xml file. Not sure why it's specified there. Let me figure out where it came from.

Reply:
It might be an old version of our IntelliSense. See this PR which removed those: https://github.com/dotnet/dotnet-api-docs/pull/212